CMakeLists.txt

```diff
@@ -37,21 +38,22 @@ if(is_big_endian)
 endif()
 
 cable_configure_compiler()
-add_compile_options(
+cable_add_compile_options(
     -Wcast-qual
     -Wcast-align
     -Wmissing-declarations
-    $<$<COMPILE_LANGUAGE:CXX>:-Wextra-semi>
-    $<$<COMPILE_LANGUAGE:CXX>:-Wold-style-cast>
```

Comment:
Does `COMPILE_LANGUAGE:CXX` means if the file is cpp or if the compiler is run in c++ mode?

Reply:
It means C++ compilation determined by CMake by the source file extension (the fact that this is the same tool for C and C++ is only  coincidence).